frontend/packages/css/typography.module.scss

```diff
@@ -1,5 +1,4 @@
-@import '@/common/styles/primitiveColors.css';
```

Comment:
let's not remove `@import 'primitiveColors.css'; ` in scope of this pr. This might cause unexpected regressions